### ISSUE TITLE
[NEW KIT] keli5's music kit

### ISSUE THREAD

**Issue by keli5** [KadTheHunter/ShulkerArchives#60]
### Kit Name
- keli5's music kit

### Author
- keli5

### Description
- A kit with music discs relabeled as other songs, with lyrics in the lore.

### Block
(e.g red_shulker_box)
- gray_shulker_box

### NBT Data 
(Paste on the line between backticks)
```
{Count:1b,id:"minecraft:gray_shulker_box",tag:{BlockEntityTag:{Items:[{Count:1b,Slot:0b,id:"minecraft:gray_stained_glass_pane",tag:{}},{Count:1b,Slot:1b,id:"minecraft:gray_stained_glass_pane",tag:{}},{Count:1b,Slot:2b,id:"minecraft:gray_stained_glass_pane",tag:{}},{Count:1b,Slot:3b,id:"minecraft:gray_stained_glass_pane",tag:{}},{Count:1b,Slot:4b,id:"minecraft:paper",tag:{display:{Name:'{"italic":false,"extra":[{"text":""},{"color":"light_purple","text":"keli5\'s music kit"}],"text":""}'}}},{Count:1b,Slot:5b,id:"minecraft:gray_stained_glass_pane",tag:{}},{Count:1b,Slot:6b,id:"minecraft:gray_stained_glass_pane",tag:{}},{Count:1b,Slot:7b,id:"minecraft:gray_stained_glass_pane",tag:{}},{Count:1b,Slot:8b,id:"minecraft:gray_stained_glass_pane",tag:{}},{Count:1b,Slot:9b,id:"minecraft:music_disc_ward",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Green Day - American Idiot"}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Don\'t wanna be an "},{"color":"red","text":"American idiot"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Don\'t want a nation under the new media"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"And can you hear the sound of hysteria"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"The subliminal "},{"color":"red","text":"mind-fuck"},{"color":"aqua","text":" America"}],"text":""}']}}},{Count:1b,Slot:10b,id:"minecraft:black_stained_glass_pane",tag:{}},{Count:1b,Slot:11b,id:"minecraft:music_disc_13",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Powerwolf - Die, Die, Crucify"}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"red","text":"Die, die, crucify"},{"color":"aqua","text":", dominus en fortuna"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Stand up and "},{"color":"red","text":"fight"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Sacrilege of doom tonight"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Reborn for regret"}],"text":""}']}}},{Count:1b,Slot:12b,id:"minecraft:black_stained_glass_pane",tag:{}},{Count:1b,Slot:13b,id:"minecraft:music_disc_chirp",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Kendrick Lamar - HUMBLE."}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Ayy, I remember syrup sandwiches and"},{"color":"red","text":" crime allowances"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Finesse a "},{"obfuscated":true,"color":"aqua","text":"words "},{"color":"aqua","text":"with some counterfeits"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Now I\'m counting this"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Parmesan where my accountant lives"}],"text":""}']}}},{Count:1b,Slot:14b,id:"minecraft:black_stained_glass_pane",tag:{}},{Count:1b,Slot:15b,id:"minecraft:music_disc_mellohi",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Kendrick Lamar - i"}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"I done been through a whole lot"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Trial, tribulation, but I know "},{"color":"dark_green","text":"God"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"red","text":"Satan"},{"color":"aqua","text":" wanna put me in a bow tie"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Pray that the holy water don\'t go dry"}],"text":""}']}}},{Count:1b,Slot:16b,id:"minecraft:black_stained_glass_pane",tag:{}},{Count:1b,Slot:17b,id:"minecraft:music_disc_far",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"The Charlie Daniels Band - The Devil Went Down to Georgia"}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Johnny, rosin up your bow and play your fiddle hard"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Cause Hell\'s broke loose in Georgia and the Devil deals the cards"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"And if you win you get this shiny fiddle made of "},{"color":"yellow","text":"gold"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"But if you lose, the Devil gets your soul"}],"text":""}']}}},{Count:1b,Slot:18b,id:"minecraft:black_stained_glass_pane",tag:{}},{Count:1b,Slot:19b,id:"minecraft:music_disc_mall",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Bon Jovi - You Give Love A Bad Name"}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"red","text":"Shot through the heart"},{"color":"aqua","text":" and you\'re to blame"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"You give love a "},{"color":"light_purple","text":"bad name"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"I play my part and you play your game"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"You give love a bad name"}],"text":""}']}}},{Count:1b,Slot:20b,id:"minecraft:black_stained_glass_pane",tag:{}},{Count:1b,Slot:21b,id:"minecraft:music_disc_blocks",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Tristam & Braken - Frame of Mind"}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"You can lift your head up to the "},{"color":"dark_aqua","text":"sky"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Take a deeper breath and give it "},{"color":"dark_aqua","text":"time"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"You can walk the path among the "},{"color":"dark_aqua","text":"lines"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"With your shattered frame of "},{"color":"dark_aqua","text":"mind"}],"text":""}']}}},{Count:1b,Slot:22b,id:"minecraft:black_stained_glass_pane",tag:{}},{Count:1b,Slot:23b,id:"minecraft:music_disc_wait",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Lil Nas X - INDUSTRY BABY"}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Baby back, ayy, couple racks, ayy"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Couple "},{"color":"yellow","text":"Grammys"},{"color":"aqua","text":" on him"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Couple "},{"color":"white","text":"plaques"},{"color":"aqua","text":", ayy"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"That\'s a fact, ayy"}],"text":""}']}}},{Count:1b,Slot:24b,id:"minecraft:black_stained_glass_pane",tag:{}},{Count:1b,Slot:25b,id:"minecraft:music_disc_stal",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"2Pac - Changes"}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Come on, come on"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"I see "},{"color":"red","text":"no changes"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Wake up in the morning and I ask myself"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Is life worth living, should I blast myself"},{"text":"?"}],"text":""}']}}},{Count:1b,Slot:26b,id:"minecraft:black_stained_glass_pane",tag:{}}]},HideFlags:127,display:{Name:'{"extra":[{"bold":false,"italic":false,"underlined":false,"strikethrough":false,"obfuscated":false,"color":"light_purple","text":"keli5\'s music kit"}],"text":""}'}}}
```

**Comment by KadTheHunter** [comment]
```
{BlockEntityTag:{Items:[{Count:1b,Slot:0b,id:"minecraft:gray_stained_glass_pane",tag:{}},{Count:1b,Slot:1b,id:"minecraft:gray_stained_glass_pane",tag:{}},{Count:1b,Slot:2b,id:"minecraft:gray_stained_glass_pane",tag:{}},{Count:1b,Slot:3b,id:"minecraft:gray_stained_glass_pane",tag:{}},{Count:1b,Slot:4b,id:"minecraft:paper",tag:{display:{Name:'{"italic":false,"extra":[{"text":""},{"color":"light_purple","text":"keli5\'s music kit"}],"text":""}'}}},{Count:1b,Slot:5b,id:"minecraft:gray_stained_glass_pane",tag:{}},{Count:1b,Slot:6b,id:"minecraft:gray_stained_glass_pane",tag:{}},{Count:1b,Slot:7b,id:"minecraft:gray_stained_glass_pane",tag:{}},{Count:1b,Slot:8b,id:"minecraft:gray_stained_glass_pane",tag:{}},{Count:1b,Slot:9b,id:"minecraft:music_disc_ward",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Green Day - American Idiot"}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Don\'t wanna be an "},{"color":"red","text":"American idiot"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Don\'t want a nation under the new media"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"And can you hear the sound of hysteria"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"The subliminal "},{"color":"red","text":"mind-fuck"},{"color":"aqua","text":" America"}],"text":""}']}}},{Count:1b,Slot:10b,id:"minecraft:black_stained_glass_pane",tag:{}},{Count:1b,Slot:11b,id:"minecraft:music_disc_13",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Powerwolf - Die, Die, Crucify"}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"red","text":"Die, die, crucify"},{"color":"aqua","text":", dominus en fortuna"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Stand up and "},{"color":"red","text":"fight"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Sacrilege of doom tonight"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Reborn for regret"}],"text":""}']}}},{Count:1b,Slot:12b,id:"minecraft:black_stained_glass_pane",tag:{}},{Count:1b,Slot:13b,id:"minecraft:music_disc_chirp",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Kendrick Lamar - HUMBLE."}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Ayy, I remember syrup sandwiches and"},{"color":"red","text":" crime allowances"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Finesse a "},{"obfuscated":true,"color":"aqua","text":"words "},{"color":"aqua","text":"with some counterfeits"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Now I\'m counting this"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Parmesan where my accountant lives"}],"text":""}']}}},{Count:1b,Slot:14b,id:"minecraft:black_stained_glass_pane",tag:{}},{Count:1b,Slot:15b,id:"minecraft:music_disc_mellohi",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Kendrick Lamar - i"}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"I done been through a whole lot"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Trial, tribulation, but I know "},{"color":"dark_green","text":"God"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"red","text":"Satan"},{"color":"aqua","text":" wanna put me in a bow tie"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Pray that the holy water don\'t go dry"}],"text":""}']}}},{Count:1b,Slot:16b,id:"minecraft:black_stained_glass_pane",tag:{}},{Count:1b,Slot:17b,id:"minecraft:music_disc_far",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"The Charlie Daniels Band - The Devil Went Down to Georgia"}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Johnny, rosin up your bow and play your fiddle hard"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Cause Hell\'s broke loose in Georgia and the Devil deals the cards"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"And if you win you get this shiny fiddle made of "},{"color":"yellow","text":"gold"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"But if you lose, the Devil gets your soul"}],"text":""}']}}},{Count:1b,Slot:18b,id:"minecraft:black_stained_glass_pane",tag:{}},{Count:1b,Slot:19b,id:"minecraft:music_disc_mall",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Bon Jovi - You Give Love A Bad Name"}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"red","text":"Shot through the heart"},{"color":"aqua","text":" and you\'re to blame"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"You give love a "},{"color":"light_purple","text":"bad name"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"I play my part and you play your game"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"You give love a bad name"}],"text":""}']}}},{Count:1b,Slot:20b,id:"minecraft:black_stained_glass_pane",tag:{}},{Count:1b,Slot:21b,id:"minecraft:music_disc_blocks",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Tristam & Braken - Frame of Mind"}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"You can lift your head up to the "},{"color":"dark_aqua","text":"sky"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Take a deeper breath and give it "},{"color":"dark_aqua","text":"time"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"You can walk the path among the "},{"color":"dark_aqua","text":"lines"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"With your shattered frame of "},{"color":"dark_aqua","text":"mind"}],"text":""}']}}},{Count:1b,Slot:22b,id:"minecraft:black_stained_glass_pane",tag:{}},{Count:1b,Slot:23b,id:"minecraft:music_disc_wait",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Lil Nas X - INDUSTRY BABY"}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Baby back, ayy, couple racks, ayy"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Couple "},{"color":"yellow","text":"Grammys"},{"color":"aqua","text":" on him"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Couple "},{"color":"white","text":"plaques"},{"color":"aqua","text":", ayy"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"That\'s a fact, ayy"}],"text":""}']}}},{Count:1b,Slot:24b,id:"minecraft:black_stained_glass_pane",tag:{}},{Count:1b,Slot:25b,id:"minecraft:music_disc_stal",tag:{HideFlags:32,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"2Pac - Changes"}],"text":""}','{"extra":[{"text":""}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Come on, come on"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"I see "},{"color":"red","text":"no changes"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Wake up in the morning and I ask myself"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"aqua","text":"Is life worth living, should I blast myself"},{"text":"?"}],"text":""}']}}},{Count:1b,Slot:26b,id:"minecraft:black_stained_glass_pane",tag:{}}]},HideFlags:127,display:{Name:'{"extra":[{"bold":false,"italic":false,"underlined":false,"strikethrough":false,"obfuscated":false,"color":"light_purple","text":"keli5\'s music kit"}],"text":""}'}}
```

Updated NBT to be only the necessary NBT (removed `{Count:1b,id:"minecraft:gray_shulker_box",tag:` and a corresponding `}` at the end)
